app/views/admin/suppliers/lead_provider_users/delete.html.erb

```diff
@@ -1,5 +1,6 @@
 <h1>Do you want to delete this user?</h1>
-<p>User: <%= @lead_provider_user.full_name %></p>
+<p class="govuk-!-font-weight-bold">User:</p>
+<p class="govuk-!-font-weight-bold">Supplier: <span class="govuk-!-font-weight-regular"><%= @lead_provider_user.full_name %></span></p>
```

Comment:
This is supposed to be the lead provider name I think, the user name goes with User

Reply:
Conversations with Shani will be much easier if you have realistic example data, "Second Lp User" doesn't look like a name, and it's confusing matters

Reply:
Just changed the names in Dummy structures

Reply:
Marking as done after Shani is happy with

`<p class="govuk-!-font-weight-bold">Supplier user: <span class="govuk-!-font-weight-regular"><%= @lead_provider_user.full_name %></span></p>`